agent/api/task/task_test.go

```diff
@@ -1135,6 +1135,12 @@ func TestTaskFromACS(t *testing.T) {
 						Region:    strptr("us-west-2"),
 					},
 				},
+				EnvironmentFiles: []*ecsacs.EnvironmentFile{
+					{
+						Value: strptr("s3://bucketName/envFile"),
+						Type:  strptr("s3"),
```

Comment:
do we need to add test for when this is not s3?

Reply:
customers wouldn't be able to register a task def for values other than s3 currently based on backend logic.. do you think that's still needed as a test on our side?